.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "flatbuffers"]
```

Comment:
I am a bit surprised that the submodule is deleted here and `find_package` is used in CMakeLists instead. @pmai why do you suggest this particular change?

Background info: I find it very convenient to use a submodule here and not have the dependency on an (possibly) existing package. It is one of the pro-points on the list for flatbuffers that this is somehow possible..

Reply:
While I also found this somewhat convenient, it has its own problems (both in nailing down the version of flatbuffers needlessly, and not making it easy to configure for different target settings), and it is not what we do for protobuf. And with todays availability of vcpkg and others for installing flatbuffers, it seems better to stay consistently out of the submodule business for these kinds of dependencies.

Reply:
Understood. On the other side, nailing down the version of flatbuffers (and possibly protobuf) could prevent some missunderstandings in combining different models / tool plugins. The flexibility to configure for different is indeed very important for our project. So if using submodules can be a problem to this point, I am ok with the change here.